From 8c0a3c70b32e936f740c0f902f250cc473af08b3 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:05:45 +0800
Subject: [PATCH] net/batman-adv/log.c: remove superfluous headers

log.c hasn't use any macro or function declared in main.h.
Thus, these files can be removed from log.c safely without
affecting the compilation of the ./net module
---
 net/batman-adv/log.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/batman-adv/log.c b/net/batman-adv/log.c
index 7a93a1e94..413485af9 100644
--- a/net/batman-adv/log.c
+++ b/net/batman-adv/log.c
@@ -5,7 +5,6 @@
  */
 
 #include "log.h"
-#include "main.h"
 
 #include <linux/stdarg.h>
 
-- 
2.25.1

